index.html

```diff
@@ -331,52 +331,52 @@ <h2>About 8-bit Games</h2>
+            <div class="testslides">
+                <div class="testImg">
+                    <img src="img/person1.png" alt="person1">
+                </div>
+                <div class="text">
+                    <p class="name"><b> John Smith </b></p>
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <p><b>Retro Game is the best!</b><br>
+                    <p>Best retro game platform ever. Just click and start playing.<br> Reminds me of my childhood. So
+                        nostalgic.</p>
+                </div>
             </div>
-        </div>
 
-        <div class="testslides">
-            <div class="testImg">
-                <img src="img/person2.png" alt="person2">
-            </div>
-            <div class="text">
-                <p class="name"><b> Karen Brown </b></p>
-                <img src="/img/star.png">
-                <img src="/img/star.png">
-                <p><b>Could be better!</b><br>
-                <p>I don't know why, but this website could be better. I mean <br> I love the games but could be better.</p>
+            <div class="testslides">
+                <div class="testImg">
+                    <img src="img/person2.png" alt="person2">
+                </div>
+                <div class="text">
+                    <p class="name"><b> Karen Brown </b></p>
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <p><b>Could be better!</b><br>
+                    <p>I don't know why, but this website could be better. I mean <br> I love the games but could be better.</p>
+                </div>
             </div>
-        </div>
 
-        <div class="testslides">
-            <div class="testImg">
-                <img src="img/person3.png" alt="person1">
-            </div>
-            <div class="text">
-                <p class="name"><b> Rudolf Klinman </b></p>
-                <img src="/img/star.png">
-                <img src="/img/star.png">
-                <img src="/img/star.png">
-                <img src="/img/star.png">
-                <img src="/img/star.png">
-                <p><b>One word! Amazing!</b><br>
-                <p>I think I cried when I played Super Mario. I hope you all <br> add more games.</p>
+            <div class="testslides">
+                <div class="testImg">
+                    <img src="img/person3.png" alt="person1">
+                </div>
+                <div class="text">
+                    <p class="name"><b> Rudolf Klinman </b></p>
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
+                    <img src="/img/star.png">
```

Comment:
lets use a component that will do this so that when you give a number it will render that much star element